docs/ugcstor.md

```diff
@@ -544,6 +544,139 @@ Note: the deletion of backup will not delete the snapshots created as part of ba
+
+```
+...
+    spec:
+      containers:
+        - name: velero
+          command:
+            - /velero
+          args:
+            - server
+          - --default-backup-storage-location=<location1>
+```
+
+2. Failed to take backup
+
+This error happens when cStor volume doesn’t have sufficient healthy replicas to create a snapshot.
+
+To check the replica status:
+```
+~$ kubectl get cstorvolume -n openebs
+NAME                                       STATUS    AGE
+pvc-2559f80b-35a0-11e9-92c5-42010a80006f   Healthy   2h
+```
+
+3. Backup PartiallyFailed
+
+If all the CVRs are healthy and snapshot creation successful then check for the following logs in `velero backup logs <BACKUP_NAME>
+
+```
+time="2020-03-19T10:47:18Z" level=warning msg="Failed to close file interface : blob (code=Unknown): MultipartUpload: upload multipart failed\n\tupload id: b18cebe2-1f10-4688-93af-9ce3eee24ba8\ncaused by: TotalPartsExceeded: exceeded total allowed configured MaxUploadParts (10000). Adjust PartSize to fit in this limit" backup=velero/daily-k8stest-backup-openebs-120d-20200319084428 cmd=/plugins/velero-blockstore-cstor logSource="/home/travis/gopath/src/github.com/openebs/velero-plugin/pkg/clouduploader/conn.go:242" pluginName=velero-blockstore-cstor
+```
+
+Above is a known issue with velero-plugin having version <= 1.8.0-velero_1.0.0.
+
+4. Debugging Restore
+
+When Velero finishes a Restore, its status changes to `Completed` regardless of whether or not there are issues during the process. The number of warnings and errors are indicated in the output columns from velero restore get:
+```
+~$ velero restore get
+NAME                BACKUP    STATUS      WARNINGS   ERRORS    CREATED                         SELECTOR
+s2-20190221142323   s2        Completed   2          1         2019-02-21 14:23:24 +0530 IST   <none>
+```
+
+To delve into the warnings and errors in more detail, you can use velero restore describe:
+```
+velero restore describe s2-20190221142323
+```
+
+The output looks like this:
+```
+Name:         s2-20190221142323
+Namespace:    velero
+Labels:       <none>
+Annotations:  <none>
+
+Backup:  s2
+
+Namespaces:
+  Included:  *
+  Excluded:  <none>
+
+Resources:
+  Included:        *
+  Excluded:        nodes, events, events.events.k8s.io, backups.velero.io, restores.velero.io
+  Cluster-scoped:  auto
+
+Namespace mappings:  <none>
+
+Label selector:  <none>
+
+Restore PVs:  true
+
+Phase:  Completed
+
+Validation errors:  <none>
+
+Warnings:
+  Velero:        <none>
+  Cluster:    <none>
+  Namespaces:
+    litmus:  not restored: persistentvolumeclaims "demo-cstor-vol-claim-ps" already exists and is different from backed up version.
+             not restored: cstorbackupdatas.openebs.io "test2" already exists and is different from backed up version.
+
+Errors:
+  Velero:        <none>
+  Cluster:  <none>
+  Namespaces: <none>
+```
+
+Errors appear for incomplete or partial restores.
```

Comment:
Need some more clarity in the first sentence.. Something like this way:
Errors appear for incomplete or partial restores.    ->  Errors usually appear for incomplete or partial restores.